apps/web/utils/actions/group.ts

```
@@ -13,216 +13,227 @@ import { findNewsletters } from "@/utils/ai/group/find-newsletters";
+        ),
+    );
+
+    await prisma.groupItem.createMany({
+      data: newItems.map((item) => ({
+        type: GroupItemType.FROM,
+        value: item,
+        groupId,
+      })),
+    });
+
+    revalidatePath(`/automation`);
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)